yt_dlp/downloader/external.py

```diff
@@ -615,6 +615,8 @@ def _call_downloader(self, tmpfilename, info_dict):
         else:
             args += ['-f', EXT_TO_OUT_FORMATS.get(ext, ext)]
 
+        args += traverse_obj(info_dict, ('downloader_options', 'ffmpeg_args_out'), default=[])
```

Comment:
Not quite the same but shorter and more explicit imo
```suggestion
        args += traverse_obj(info_dict, ('downloader_options', 'ffmpeg_args_out', ...))
```

Reply:
I was just mirroring L494, which I would also wanna change

Reply:
It's old code back from when `...` didn't guarantee list. You can change